src/generate/handlebarsAmfHelpers.ts

```diff
@@ -77,12 +79,13 @@ export const getReturnPayloadType = (
 };
 
 /**
- * Get data type of a property
+ * Get data type of a property.
+ *
+ * @param property - An AMF property
  *
- * @param property - instance of model.domain.PropertyShape
- * @returns data type if defined in the property otherwise returns a default type
+ * @returns data type, if defined in the property, the default type otherwise
  */
-export const getPropertyDataType = (
+export const getTypeFromProperty = (
```

Comment:
How does a property differ from a parameter?

Reply:
The only difference is the kind of AMF element they operate on.
model.domain.PropertyShape vs model.domain.Parameter